### PR TITLE
V1.3 Release Notes

### PR DESCRIPTION
**Pull request by clwongOSIsoft**
Updated the release notes for version 1.3

### CODE REVIEW

File: content/release-notes/release-notes.md [hunk 1]
```diff
@@ -16,21 +16,15 @@ For more information, see [EdgeCmd utility](xref:index).
...
-* Added support for data source discovery.
-* Added *cancel* and *resume* keywords for history recoveries and *cancel* for discoveries.
-* Added support for CSV import and export of complex types. 
-* Improved help messages.
-* Upgraded runtime to .NET 5.0.
-* Enhanced Windows installation experience.
-* Added EdgeCmd to the PATH on Windows.
+* Added support for secret management.
+* Added support for retrieving failover state.
+* Updated Windows and Linux installers to align with renaming work.
```
Comment (aveva-jhwang) [inline content/release-notes/release-notes.md:27]: I would rephrase it to "Updated Windows and Linux installers to reflect new product name."

File: content/release-notes/release-notes.md [hunk 2]
```diff
@@ -16,21 +16,15 @@ For more information, see [EdgeCmd utility](xref:index).
...
 
-* Added support for data source discovery.
-* Added *cancel* and *resume* keywords for history recoveries and *cancel* for discoveries.
-* Added support for CSV import and export of complex types. 
-* Improved help messages.
-* Upgraded runtime to .NET 5.0.
-* Enhanced Windows installation experience.
-* Added EdgeCmd to the PATH on Windows.
+* Added support for secret management.
+* Added support for retrieving failover state.
```
Comment (aveva-jhwang) [inline content/release-notes/release-notes.md:26]: Is failover state retrieval the only update related to this feature? I thought there were other things related to failover configuration that were added. If not, this looks fine (I may be confusing it with other work related to Victoria).

**Review by aveva-jhwang — COMMENTED** [review]
Added a couple of comments under specific lines in the notes.

File: content/release-notes/release-notes.md (hunk 2 above)
Comment (clwongOSIsoft) [reply content/release-notes/release-notes.md:26]: There was no additional work done for failover configuration, but I think it might be beneficial to mention that failover configuration can be done through EdgeCmd because there could be confusion around it since it wasn't mentioned before in previous release notes.
